symengine/symengine_rcp.h

```diff
@@ -39,8 +39,8 @@ class Ptr {
     inline Ptr(const Ptr<T>& ptr) : ptr_(ptr.ptr_) {}
     template<class T2> inline Ptr(const Ptr<T2>& ptr) : ptr_(ptr.get()) {}
     Ptr<T>& operator=(const Ptr<T>& ptr) { ptr_ = ptr.get(); return *this; }
-    inline Ptr(Ptr&&) = default;
-    Ptr<T>& operator=(Ptr&&) = default;
+//    inline Ptr(Ptr&&) = default;
+//    Ptr<T>& operator=(Ptr&&) = default;
```

Comment:
This however, unlike https://github.com/sympy/symengine/pull/536/files#r34785213, is a big deal. I don't know what the problem is, if Intel doesn't support the `default` keyword, or how to workaround it.